### ISSUE TITLE
Please add instructions to install & configure with pylama.

### ISSUE THREAD

**Issue by alvassin** [PyCQA/flake8-import-order#157]
Not clear how to start using this plugin with pylama.
Can it be somehow integrated via pylama.ini?

**Comment by alvassin** [comment]
Please help :(

**Comment by pgjones** [comment]
Does something like this work in the setup.cfg?
```
[pylama]
application-import-names = quart, tests                                                                                                                                                                     
ignore = E252, W503, W504                                                                                                                                                                                   
import-order-style = smarkets                                                                                                                                                                               
max_line_length = 100 
```

**Comment by alvassin** [comment]
I added this linter to pylama as following:
```
entry_points = {
    'pylama.linter': [
        'imports_order = flake8_import_order.pylama_linter:Linter'
    ]
},
```

Also, i configured pylama using pylama.ini:
```
[pylama]
linters=pycodestyle,pyflakes,mccabe,imports_order

[pylama:imports_order]
application-import-names=myapp
import-order-style = smarkets
```

The problem is that pylama does not detect imports from `myapp` as local ones. Moreover, i do not see [`application_import_names` in `self.options`](https://github.com/PyCQA/flake8-import-order/blob/master/flake8_import_order/checker.py#L62) .

I can see that `self.options` contains the following:
```
{
    'code': '...',
    'ignore': set(),
    'select': set(),
    'params': {
        'application-import-names': 'myapp',
        'import-order-style': 'smarkets',
    },
    'import_order_style': Entrypoint('cryptography = flake8_import_order.styles:Cryptography')
}
```

**Comment by alvassin** [comment]
Can you please advice, what is going wrong?

**Comment by pgjones** [comment]
Have you tried putting this in the `pylama.ini`?
```
[pylama:imports_order]
application_import_names=myapp
import_order_style = smarkets
```

**Comment by pgjones** [comment]
I'm going to assumed the above worked, please reopen if not.
